fhir-path/src/main/java/com/ibm/fhir/path/util/FHIRPathUtil.java

```diff
@@ -1238,4 +1238,21 @@ public static com.ibm.fhir.model.type.String getDisplay(FHIRPathTree tree, FHIRP
         }
         return null;
     }
+    
+    /**
+     * returns true if the node is a String or if FHIR element can be automatically converted to System.String 
+     * https://www.hl7.org/fhir/fhirpath.html#types
+     * @param node
+     * @return boolean
+     */
+    public static boolean isStringSubType(FHIRPathNode node) {
```

Comment:
One thing I didn't think of on the call... we should be clear about whether this should return true or false for FHIRPathNode of type SYSTEM_STRING

Options:
A. add this type to the list of types we return true for; or
B. change the method signature to `isFHIRStringSubType(FHIRPathElementNode)` to make it clear this only operates over FHIRPathNode that wrap FHIR elements (not primitive values)

Reply:
Handled the above review comment with Option A.